<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoktorLoginWIndow</class>
 <widget class="QMainWindow" name="DoktorLoginWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>321</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="doktorLoginLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Doktor:</string>
    </property>
   </widget>
   <widget class="QLabel" name="doktorLoginIme">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Korisničko Ime:</string>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="loginDoktorUnosIme">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QKeySequenceEdit" name="loginDoktorUnosSifra">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="doktorLoginSifra">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Šifra:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doktorLogin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Uloguj se</string>
    </property>
   </widget>
   <widget class="QPushButton" name="doktorRegistruj">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Registruj se</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
